resources/assets/icons/server-lock.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><g clip-path="url(#clip0_1647_41977)" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" fill="none"><path d="M4.66 4.3a.3.3 0 100 .601.3.3 0 000-.6v0zm0 7.2a.3.3 0 100 .6.3.3 0 000-.6v0z" stroke-dasharray="4 4"/><path d="M12.456 8.2H4.657a3.6 3.6 0 110-7.2h9.6c2.756 0 4.792 3.1 2.803 6"/><path d="M9.457 15.399h-4.8a3.6 3.6 0 110-7.2h7.799M8.855 4.6h5.4m-5.4 7.2h1.2m1.8 2.4a1.2 1.2 0 011.2-1.2h4.8a1.2 1.2 0 011.2 1.2v3.6a1.2 1.2 0 01-1.2 1.2h-4.8a1.2 1.2 0 01-1.2-1.2v-3.6zm3.6-4.8a2.4 2.4 0 00-2.4 2.4V13h4.8v-1.204a2.4 2.4 0 00-2.4-2.397v0z"/><path d="M15.457 15.723a.3.3 0 100 .6.3.3 0 000-.6v0z" stroke-dasharray="4 4"/></g><defs><clipPath id="clip0_1647_41977"><path fill="#fff" d="M0 0h20v20H0z"/></clipPath></defs></svg>
```

Comment:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><g stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" fill="none"><path d="M4.66 4.3a.3.3 0 1 0 0 .601.3.3 0 0 0 0-.6v0zm0 7.2a.3.3 0 1 0 0 .6.3.3 0 0 0 0-.6v0z" stroke-dasharray="4 4"/><path d="M12.456 8.2H4.657a3.6 3.6 0 1 1 0-7.2h9.6c2.756 0 4.792 3.1 2.803 6"/><path d="M9.457 15.399h-4.8a3.6 3.6 0 1 1 0-7.2h7.799M8.855 4.6h5.4m-5.4 7.2h1.2m1.8 2.4a1.2 1.2 0 0 1 1.2-1.2h4.8a1.2 1.2 0 0 1 1.2 1.2v3.6a1.2 1.2 0 0 1-1.2 1.2h-4.8a1.2 1.2 0 0 1-1.2-1.2v-3.6zm3.6-4.8a2.4 2.4 0 0 0-2.4 2.4V13h4.8v-1.204a2.4 2.4 0 0 0-2.4-2.397v0z"/><path d="M15.457 15.723a.3.3 0 1 0 0 .6.3.3 0 0 0 0-.6v0z" stroke-dasharray="4 4"/></g></svg>
```

Reply:
Did you do something special with this @leMaur? Since I got mine from Figma :thinking: was it a different tool you used?

Reply:
@alexbarnsley Icons are made with Adobe Illustrators and working with vector graphics may add some clip paths. Most of the times, those paths do nothing on the icon itself and we can safely remove them.
I usually open the original icon on https://jakearchibald.github.io/svgomg/ and the modified one in another tab (always on svgomg). If no visual differences, I grab the new code otherwise I ask Oleg to remove the clip-path.

Reply:
:+1: thanks Maui